data/sles4sap/qe_sap_deployment/qesap_aws.yaml

```diff
@@ -12,6 +12,7 @@ terraform:
     hana_instancetype: "%HANA_INSTANCE_TYPE%"
     hana_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
     iscsi_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
+    vpc_address_range: "10.2.0.0/19"
```

Comment:
I don't like hard-coded IP range like this. It need to be calculated or taken from the Terraform  of the deployment.